MediaAdminBundle/Tests/FileUtils/AbstractFileUtilsManager.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminBundle\Tests\FileUtils;
+
+use Phake;
+use Imagick;
+
+/**
+ * Class AbstractFileUtilsManager
+ */
+abstract class AbstractFileUtilsManager extends \PHPUnit_Framework_TestCase
+{
+    protected $fixturesPath;
+
+    protected $generatedFiles = array();
+
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        $this->fixturesPath = __DIR__ . '/../Fixtures/';
+    }
+
+    /**
+     * Check if a file has been correctly generated and tag it to be deleted on tearDown
+     * 
+     * @param string $expectedFile
+     * @param string $generatedFile
+     */
+    protected function assertFileCorrectlyGenerated($expectedFile, $generatedFile)
+    {
+        $this->generatedFiles[] = $generatedFile;
+
+        $this->assertFileExists($generatedFile);
+//        $this->assertFileEquals($expectedFile, $generatedFile);
```

Comment:
@nicolasThal : this was made intentionnally to talk about it :) 
Testing the image size is worst than really testing the image itself. But as you have already see it previously, the image generation on travis don't produce the same image as locally (maybe meta-info change).

So do you have a better way to compare 2 images ?